docs/backend/data-update-scripts.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Data Update Scripts
+---
+
+## What are Data Update Scripts?
+
+Data Update Scripts were introduced in
+[this PR](https://github.com/thepracticaldev/dev.to/pull/6025) and allow us to
+run any data updates we might need. For example, if we added a column to the
+database and then wanted to backfill that column with data, rather than going
+and manually doing it in a console, we would use a DataUpdateScript. Another
+example might be adding a new attribute to Elasticsearch. We could then use a
+DataUpdateScript to reindex all of our models.
+
+## How it works
+
+First off, we added a
+[DataUpdateScript model](https://github.com/thepracticaldev/dev.to/blob/master/app/models/data_update_script.rb)
+to Rails and a corresponding database table. This table is what we use to keep
+track of what scripts have been run and which ones have not/still need to be.
+
+To create a script you can use our custom Rails generator:
+
+```
+rails generate data_update BackfillColumnForArticles
+```
+
+This will create a simple Ruby class like below and all you have to do is fill
+in the code it will run.
+
+```ruby
+module DataUpdateScripts
+  class BackfillColumnForArticles
+    def run
+      # Place your data update logic here
+      # Make sure your code is idempotent and can be run safely
+      # multiple times at any time
+    end
+  end
+end
+```
+
+Once your script is in place then you can either run `rake data_updates:run`
```

Comment:
s/rake/rails ? I know sometimes we use `rails` as the runner and other times we use `rake` as the runner in the documentation and in the code